scraper.rb

```diff
@@ -10,5 +10,5 @@
 members = Members.new(members_url).to_h[:members_of_the_lower_house]
 
 members.map do |member|
```

Comment:
I realise it's a pre-existing condition, but rather than calling `map` in a void context like this, this should probably also be `.each`, and as the thing you're changing is the only thing that happens inside that loop, I think fixing it here makes sense.